src/power_grid_model_io/config/excel/vision_en.yaml

```diff
@@ -549,50 +571,50 @@ grid:
       status: Switch state
       type: 0
       p_specified:
-        power_grid_model_io.filters.multiply:
+        multiply:
           - min:
-             - Pnom
-             - Inverter.Pnom | Inverter.Snom
+              - Pnom
+              - Inverter.Pnom | Inverter.Snom
           - Scaling
       q_specified:
-        power_grid_model_io.filters.multiply:
-          - power_grid_model_io.filters.phase_to_phase.reactive_power:
-             - min:
-                 - Pnom
-                 - Inverter.Pnom | Inverter.Snom
-             - Inverter.cos phi
+        multiply:
+          - power_grid_model_io.functions.phase_to_phase.reactive_power:
+              p:
+                min:
+                  - Pnom
+                  - Inverter.Pnom | Inverter.Snom
+              cos_phi: Inverter.cos phi
           - Scaling
       extra:
         - ID
         - Name
 units:
-  A:
+  A: null
   F:
-    µF: 0.000_001
+    µF: 0.000001
```

Comment:
Underscores are gone and values changed to int

Reply:
whoops... thnx, just fixed it.